app/views/notifications/shared/_comment_box.html.erb

```diff
@@ -68,20 +68,18 @@
           </a>
         </div>
 
-        <% cache "comment-box-comment-reply-#{@last_user_comment}-#{json_data['comment']['updated_at']}-#{json_data['comment']['id']}" do %>
```

Comment:
Any idea why we had this cache here in the first place? Does Github history give us any hints?

Reply:
It was basically me not really understanding how cache works... With my notifications update yesterday I move some cheese around and I though that comment form must be inside that cache and I simply duplicated existing one...

https://github.com/forem/forem/pull/11084/files#diff-c277bcefb301cd08fba69e9c0056badbff08deb289fe2b4651020f64317c9820L37-L63

^ as you can see the form was never part of the cache and we've never had two same cache boxes (?) here.

Reply:
Oooooh I see, you added a second cache where we didn't have one before 👍  mystery solved!